flaml/nlp/autotransformers.py

```diff
@@ -433,10 +539,12 @@ def _verify_init_config(self,
         for key in custom_hpo_args.keys():
             if key == "points_to_evaluate":
                 for each_init_config in custom_hpo_args[key]:
-                    for each_hp in each_init_config.keys():
-                        assert each_hp in self._search_space_hpo.keys(), \
-                            "points_to_evaluate hp must be within the search space"
+                    for each_hp in list(each_init_config.keys()):
+                        if each_hp not in self._search_space_hpo.keys():
+                            del each_init_config[each_hp]
+                        print("{} is not in the search space, deleting from init config".format(each_hp))
```

Comment:
why do we allow keys not in the search space?

Reply:
Not allowing, we are deleting it.

Reply:
use logger.warning?